x-pack/plugins/apm/server/lib/service_map/get_service_map.ts

```diff
@@ -77,7 +73,9 @@ async function getConnectionData({
 async function getServicesData(options: IEnvOptions) {
   const { setup } = options;
 
-  const projection = getServicesProjection({ setup });
+  const projection = getServicesProjection({
+    setup: { ...setup, uiFiltersES: [] }
```

Comment:
Could still be problematic that we don't pass `environment` to the service projection but I don't think you need to do anything about that in this PR